CONTRIBUTORS.md

```diff
@@ -37,14 +38,19 @@ The following individuals have contributed code to Galaxy:
 * Michael R. Crusoe <crusoe@ucdavis.edu>
 * Gianmauro Cuccuru <gmauro@crs4.it>
 * Frederik Delaere <frederik.delaere@gmail.com>
+* Matthias Desmet <matthias.desmet@ugent.be>
 * Olivia Doppelt <olivia.doppelt@pasteur.fr>
+* Shane Dowling <shane@shanedowling.com>
 * John Duddy <jduddy@illumina.com>
 * Carl Eberhard <carlfeberhard@gmail.com>
+* Mark Einon <mark.einon@gmail.com>
 * Kyle Ellrott <kellrott@gmail.com> <kellrott@soe.ucsc.edu>
 * Eric Enns <eric.enns@gmail.com>
+* fescudie <fescudie@toulouse.inra.fr>
 * Dorine Francheteau <dorine@bx.psu.edu>
 * Jaime Frey <jfrey@cs.wisc.edu>
-* Carrie Ganote <cganote@iu.edu>
+* Ryan Golhar <ngsbioinformatics@gmail.com>
+* Carie Genote <cganote@iu.edu>
```

Comment:
Hrmm.  This wasn't actually changed in this PR, and was incorrect before it, and has since been fixed in -dev.  I'm not sure why exactly github is showing it as a diff, but I'll make sure it doesn't get applied as such.